SPECS-EXTENDED/fonts-rpm-macros/fonts-rpm-macros.spec

```diff
@@ -0,0 +1,240 @@
+Vendor:         Microsoft Corporation
```

Comment:
Can we please attach a full build pipeline of both ARM/AMD with it.
Just to ensure we are not breaking anything :)

Reply:
Triggered a full build. BuildId: 372290

Reply:
AMD64 BuildRPMs complete. ARM64 BuildRPMs failed on doxigen, which is not relevant to this PR. Retriggered.